core/application/app_configuration.hpp

```diff
@@ -79,6 +79,21 @@ namespace kagome::application {
      */
     virtual uint16_t p2pPort() const = 0;
 
+    /**
+     * @return number of outgoing connections we're trying to maintain
+     */
+    virtual uint16_t outPeers() const = 0;
```

Comment:
Because size_t is not platform-specific. And we need something fixed for CLI parsing. Since u16 is enough, there's no reason to widen it up to size_t later. That's my line of thought.